sycl/plugins/hip/pi_hip.cpp

```diff
@@ -5410,6 +5410,7 @@ pi_result piPluginInit(pi_plugin *PluginInit) {
   // Queue
   _PI_CL(piQueueCreate, hip_piQueueCreate)
   _PI_CL(piextQueueCreate, hip_piextQueueCreate)
+  _PI_CL(piextQueueCreate2, hip_piextQueueCreate)
```

Comment:
why don't you need to "die" for other "2" APIs?

Reply:
piQueueCreate/piQueueCreate2 are supposed to be used for SYCL queues, and all plugins will need to support them.
The make_queue2 and get_native2 are only needed in the L0 plugin, so they are left out of this plugin.

Reply:
I think you should still "die" in those other plugins as unsupported

Reply:
OK, done.